### DISCUSSION TITLE
rtl8822bu

### DISCUSSION THREAD

**Discussion by te1lmEY** [chris1111/Wireless-USB-OC-Big-Sur-Adapter#281] [Q&A] [answered]
Please see how to support rtl8822bu network card

**Comment by chris1111** [comment, marked as answer]
RTL8822BU is Not support, Only RTL8812BU

**Reply by te1lmEY** [reply]
Do you think there is a product that supports mac os 12.5 network card and bluetooth receiver? Or only the network card function at most

**Reply by chris1111** [reply]
> Do you think there is a product that supports mac os 12.5 network card and bluetooth receiver? Or only the network card function at most

[Intel Wifi ](https://github.com/OpenIntelWireless/itlwm)or Native Broadcom
